opentracing-api/src/main/java/io/opentracing/SpanScheduler.java

```diff
@@ -0,0 +1,87 @@
+package io.opentracing;
+
+/**
+ * Scheduler allows an existing (possibly thread-local-aware) execution context provider to also manage an
+ * actively-scheduled OpenTracing Span.
+ * <p>
+ * In any execution context (or any thread, etc), there is at most one "active" Span primarily responsible for the
+ * work accomplished by the surrounding application code. That active Span may be accessed via the
+ * {@link Scheduler#active()} method. If the application needs to defer work that should be part of the same
+ * Span, the Scheduler provides a {@link Scheduler#captureActive()} method that returns a
+ * {@link Continuation}; this activation context may be used to re-activate and deactivate the
+ * captured Span in that other asynchronous executor and/or thread.
+ */
+public interface Scheduler {
```

Comment:
@mabn I did actually think about that. There are two related problems...

1. The name is already spoken for, albeit only for the MockTracer: https://github.com/opentracing/opentracing-java/blob/master/opentracing-mock/src/main/java/io/opentracing/mock/MockTracer.java#L86
2. IMO "Propagation" could be easily misunderstood to refer to cross-process propagation (a la inject/extract). I like "Scheduler" because it's unambiguous in that regard, though it's perhaps misleading... after all, it doesn't actually schedule anything. `/shrug`